ui/src/app/workflows/components/resubmit-workflow-panel.tsx

```diff
@@ -0,0 +1,106 @@
+                        <div className='columns small-9'>
+                            <Checkbox checked={this.state.overrideParameters} onChange={overrideParameters => this.setState({overrideParameters})} />
+                        </div>
+                    </div>
+
+                    {this.state.overrideParameters && (
+                        <div key='parameters' style={{marginBottom: 25}}>
+                            <label>Parameters</label>
+                            {this.state.workflowParameters.length > 0 && (
+                                <ParametersInput parameters={this.state.workflowParameters} onChange={workflowParameters => this.setState({workflowParameters})} />
+                            )}
+                            {this.state.workflowParameters.length === 0 ? (
+                                <>
+                                    <br />
+                                    <label>No parameters</label>
+                                </>
+                            ) : (
+                                <></>
+                            )}
+                        </div>
+                    )}
+
+                    <div key='memoized' style={{marginBottom: 25}}>
+                        <label>Memoized</label>
+                        <div className='columns small-9'>
+                            <Checkbox checked={this.state.memoized} onChange={memoized => this.setState({memoized})} />
+                        </div>
+                    </div>
+
+                    {this.state.overrideParameters && this.state.memoized && (
+                        <div key='warning-override-with-memoized'>
+                            <i className='fa fa-exclamation-triangle' style={{color: '#f4c030'}} />
+                            Overriding parameters on memoized submitted workflows may have unexpected results.
+                        </div>
+                    )}
+
+                    <div key='resubmit'>
+                        <button onClick={() => this.submit()} className='argo-button argo-button--base' disabled={this.state.isSubmitting}>
+                            <i className='fa fa-plus' /> {this.state.isSubmitting ? 'Loading...' : 'Resubmit'}
+                        </button>
+                    </div>
+                </div>
+            </>
+        );
+    }
+
+    private submit() {
+        this.setState({isSubmitting: true});
+        const parameters: ResubmitOpts['parameters'] = this.state.overrideParameters
+            ? [...this.state.workflowParameters.filter(p => Utils.getValueFromParameter(p) !== undefined).map(p => p.name + '=' + Utils.getValueFromParameter(p))]
+            : [];
+        const opts: ResubmitOpts = {
+            parameters,
+            memoized: this.state.memoized
+        };
+
+        services.workflows
+            .resubmit(this.props.workflow.metadata.name, this.props.workflow.metadata.namespace, opts)
```

Comment:
Could you check if this works for archived workflow as well?

Reply:
This works basically (please see PR description), but I noticed an edge case where this logic will not work where there is more than one workflow that has the same name and namespace.
Could you comment on this issue #11371?